vnext/Microsoft.ReactNative/Modules/AppStateModule.cpp

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
>All rights reserved. [](start = 40, length = 20)

I thought you started to use the new copyright headers :)

Reply:
Whats the new copyright headers?

Reply:
Microsoft says that we must not use the 'All rights reserved.' anymore for our open source MIT-licensed projects. You did it in the .h files.
See: [https://github.com/microsoft/react-native-windows/issues/3933](https://github.com/microsoft/react-native-windows/issues/3933)

---
In reply to: [429371687](https://github.com/microsoft/react-native-windows/pull/4991#discussion_r429371687) [](ancestors = 429371687)